### PR TITLE
Updated job copy

### PR DESCRIPTION
**Pull request by 409H**
* Updated jobs view copy as per Aug2020 document

### CODE REVIEW

**Comment by 409H** [comment]
@vilsbole do not merge until George has signed off pls

**Review by Balozar — APPROVED** [review]
Approved

**Review by Balozar — COMMENTED** [review]
Approved.

**Review by vilsbole — APPROVED** [review]

**Pull request merged**
